### ISSUE TITLE
how can I merge the new training data with the old data in yolov3 

### ISSUE THREAD

**Issue by JoshaPan** [AlexeyAB/darknet#4642]
If i train my yolov3 on k classes and get the weights generated. Now, if I want to add one more class to it .How to do it

Can i train it on k+1 classes just by using weights of k classes, and images of the k+1 th class.

**Comment by tom2002965** [comment]
In my opinion, Just edit classes in [yolo] layer in cfg-file.
And start training with previous weights!
If no any problems, I guess it converge fast!

**Comment by JoshaPan** [comment]
you mean that if the previous version of the hdf5 file only have 'face' class, and the class num in cfg file is 1, I just edit the num classes=2 and add one class(name 'person') to coco-classes.txt
then  start training with previous weights? right?

**Comment by tom2002965** [comment]
Yes, but actually you also check whether two classes label in your training dataset.

**Comment by JoshaPan** [comment]
thanks for your reply
I'm using the suggestions you gave to train new data now!

**Comment by JoshaPan** [comment]
hi @tom2002965 

I had used the suggestions you gave to train the new data, but I was encountered some problems as follow:

I have an previous weights named 'face', and I use this hdf5 file to train a new data, name:Joshua to let the computer not just can tell faces in picture, but also can tell Joshua in this picture

the content of the coco-classes.txt are 'face' & 'Joshua'
the number of the classes in cfg-file is '2'

when I use the new data to predict in the picture, it just show face information
the joshua information is gone

do you know why?

**Comment by JoshaPan** [comment]
you have any idea?

**Comment by tom2002965** [comment]
Maybe you could use the training data in **Joshua** to predict(test).
If show face, I think the setting is wrong.
If **Joshua**, the training is not complete. Try to train more epochs.

**Comment by JoshaPan** [comment]
when I use webcam to predict real time image
it will show face when the order of content in coco-clases is face, Joshua
and it will show 'joshua' when the order of content in coco-clases is Joshua , face

do you have any idea to solve it?
